File: eAnimalShelter/UI/eanimalshelter_desktop/lib/models/requests/announcement_insert_request.dart
class AnnouncementInsertRequest {
  String title;
  String content;
  String? imageUrl;

  AnnouncementInsertRequest({
    required this.title,
    required this.content,
    this.imageUrl,
  });

  Map<String, dynamic> toJson() {
    return {
      "title": title,
      "content": content,
      "imageUrl": imageUrl,
    };
  }
}

File: eAnimalShelter/eAnimalShelter.Model/Requests/AnnouncementInsertRequest.cs
namespace eAnimalShelter.Model.Requests
{
    public class AnnouncementInsertRequest
    {
        public string Title { get; set; } = string.Empty;

        public string Content { get; set; } = string.Empty;

        public string? ImageUrl { get; set; }

    }
}
